docs/lwnode-modules.md

```diff
@@ -14,6 +24,9 @@ For example,
 const gmainLoop = require('gmain-loop');
 ```
 
+To include a module in your app without installing the app, copy the built `<module_name>.node` file from `<lwnode_project_path>/out/modules/tizen`.
```

Comment:
`the modules` looks correct instead of `the app`.

```suggestion
To include a module in your app without installing the modules globally, copy the built `<module_name>.node` file from `<lwnode_project_path>/out/modules/tizen`.
```

Reply:
I fixed it.